src/NUnitFramework/framework/Internal/Tests/Test.cs

```diff
@@ -324,6 +324,14 @@ public void ApplyAttributesToTest(ICustomAttributeProvider provider)
         }
 #endif
 
+        /// <summary>
+        /// Clear reference to TestFixture
+        /// </summary>
+        internal void ClearTestFixture()
+        {
+            Fixture = null;
+        }
```

Comment:
As with the TestObject, Fixture also has a public setter, used elsewhere. See general comment.